src/reducers/ui-state-updaters.js

```diff
@@ -722,6 +723,22 @@ export const setLocaleUpdater = (state, {payload: {locale}}) => ({
   locale
 });
 
+/** TODO do I need state? And update docs
```

Comment:
I believe you’re defining a reducer here. If that’s true you do need the “state” code here since all other state should be passed forward untouched by a reducer function.

Reply:
👍

Reply:
One thing I couldn't figure out from the docs is what  @returns nextState is. The modal doesn't return anything but none of the other examples say that it doesn't return anything